### ISSUE TITLE
`\r` is not visible even with `--nl` mode

### ISSUE THREAD

**Issue by sobolevn** [lensvol/tokelor#3]
`tokelor` does not show `\r` tokens. I found it during working on
- https://github.com/wemake-services/wemake-python-styleguide/pull/1171
- https://github.com/wemake-services/wemake-python-styleguide/issues/1111

Reproduction:

1. `echo 'print(1)\r\nprint(2)' > ex.py`
2. `tokelor --bare --nl ex.py`

Output:

```
» tokelor --bare --nl ex.py
    1: [   NAME    ] |   0 - 5   | print(1)
                                   ^^^^^
    1: [    OP     ] |   5 - 6   | print(1)
                                        ^
    1: [  NUMBER   ] |   6 - 7   | print(1)
                                         ^
    1: [    OP     ] |   7 - 8   | print(1)
                                          ^
    1: [  NEWLINE  ] |   8 - 9   | print(1)\n
                                           ^^
    2: [   NAME    ] |   0 - 5   | print(2)
                                   ^^^^^
    2: [    OP     ] |   5 - 6   | print(2)
                                        ^
    2: [  NUMBER   ] |   6 - 7   | print(2)
                                         ^
    2: [    OP     ] |   7 - 8   | print(2)
                                          ^
    2: [  NEWLINE  ] |   8 - 9   | print(2)\n
                                           ^^
    3: [ ENDMARKER ] |   0 - 0   | <EOF>
                                   ^^^^^
```

As you can see `\r` is not showed. This can be critical in some cases.

**Comment by lensvol** [comment]
Fixed in 68ec257131363c34fa046816631062d27037af56 and 68ec257131363c34fa046816631062d27037af56, available as a part of [release 0.1.6](https://pypi.org/project/tokelor/0.1.6/).

**Comment by sobolevn** [comment]
Thank you! 👍
